pymc/model_graph.py

```diff
@@ -125,14 +125,17 @@ def make_compute_graph(
 
         return input_map
 
-    def _make_node(self, var_name, graph, *, formatting: str = "plain"):
-        """Attaches the given variable to a graphviz Digraph"""
+    def _make_node(self, var_name, graph, *, nx=False, cluster=False, formatting: str = "plain"):
```

Comment:
`formatting` is unused here, right? (not from your change, just observing)

Reply:
> `formatting` is unused here, right? (not from your change, just observing)

Yes not mine and I don't think its used.  I think it was because you can pass Tex math and things?

Reply:
yeah, looks like it -- this is low-hanging fruit for a future 'tidying' PR, but doesn't need any action here.